app/coffee/DocManager.coffee

```diff
@@ -10,7 +10,9 @@ module.exports = DocManager =
 	# collection (which is all that this collection contains). In future, we should
 	# migrate this version property to be part of the docs collection, to guarantee
 	# consitency between lines and version when writing/reading, and for a simpler schema.
-	getDoc: (project_id, doc_id, filter = { version: false }, callback = (error, doc) ->) ->
+	getDoc: (project_id, doc_id, filter = { version: false, inS3:true}, callback = (error, doc) ->) ->
+		if filter? and !filter.inS3?
+			filter.inS3 = true
```

Comment:
thought about it more and I like my implementation, inS3 is used by this function, not the function calling it. inS3 is always needed for this call and so should be a default